### PR TITLE
🧮 Aggregate channel payments

### PR DESCRIPTION
**Pull request by thesan**
- #211

### CODE REVIEW

File: schema/channels.graphql [hunk 1]
```diff
@@ -60,7 +60,10 @@ type Channel @entity {
   totalVideosCreated: Int!
 
   "Cumulative rewards claimed by this channel"
-  cumulativeRewardClaimed: BigInt
+  cumulativeRewardClaimed: BigInt!
+
+  "Cumulative rewards paid to this channel"
+  cumulativeRewardPaid: BigInt!
```
Comment (ignazio-bovo) [inline schema/channels.graphql:66]: I think it is hard now to know the difference between reward claimed vs paid.
```suggestion
  cumulativeReward: BigInt!
```
From what I understand we want to track `rewardClaimed + membershipMetadataPayment` 
So I think this is a more appropriate

File: src/mappings/content/channel.ts [hunk 2]
```diff
@@ -56,6 +56,8 @@ export async function processChannelCreatedEvent({
     followsNum,
     videoViewsNum: 0,
     totalVideosCreated: 0,
+    cumulativeRewardClaimed: 0n,
+    cumulativeRewardPaid: 0n,
```
Comment (ignazio-bovo) [inline src/mappings/content/channel.ts:60]:
```suggestion
    cumulativeReward: 0n,
```

File: src/mappings/content/metadata.ts [hunk 3]
```diff
@@ -647,6 +647,8 @@ export async function processChannelPaymentFromMember(
     }),
   })
 
+  channel.cumulativeRewardPaid += amount
```
Comment (ignazio-bovo) [inline src/mappings/content/metadata.ts:650]:
```suggestion
  channel.cumulativeReward += amount
```

File: schema/channels.graphql (hunk 1 above)
Comment (ignazio-bovo) [reply schema/channels.graphql:66]: Alternatively you could just keep an extra `cumulativeRewardFromMembers` and then on atlas you could rank by `cumulativeRewardClaimed + cumulativeRewardFromMembers`

**Review by ignazio-bovo — CHANGES_REQUESTED** [review]
Looks ok, I have just left small suggestions

**Comment by thesan** [comment]
@ignazio-bovo I renamed the field but I don't understand why after I (drop the db and) regenerate the migration the db field is still called `cumulative_reward_paid` :confused:

**Comment by ignazio-bovo** [comment]
Also please add the migration for the `NextEntityId` in the `src/utils/offchainState`
```typescript
  private globalCountersMigration = {
    // destination version : [global counters names]
    '3.0.1': ['Account'],
    '3.0.2': ['Account'],
    '3.0.3': ['Account'],
    '3.0.4': ['Account'],
  }
    ```

**Review by ignazio-bovo — APPROVED** [review]
LGTM

**Pull request merged**